__tests__/app.test.js

```diff
@@ -213,131 +213,289 @@ describe('PATCH commands', () => {
   
 });
 
-describe('Error handling', () => {
-    test("GET followed by an invalid endpoing should return a 404 Not Found error ", () => {
-      return request(app)
-        .get("/api/bananas")
-        .expect(404)
-        .then(({body}) => {
-         expect(body).toEqual({"message" : "Invalid path provided - please try again"});
+describe('Get /api/reviews Queries', () => {
+    test('200: should return the reviews with the category dexterity', () => {
+        return request(app).get("/api/reviews?category=dexterity")
+        .expect(200)
+        .then(({body})=>{
+            const reviews = body.reviews;
+          
+            expect(reviews).toHaveLength(1);
+            expect(reviews[0]).toEqual({
```

Comment:
Please check that all of the reviews have the expected category using something like a ForEach (you don't have to check all of the other keys though).

Reply:
I just realised that you're testing for each individual category. You should only have one test for "only returns reviews for the category passed", then use one of the categories to prove this. What if more categories are added later? Would you have to write a test for each one? That would be really labour intensive and unnecessary as we would already know that this functionality works. Also doing it this way is not very DRY.